static/css/index.css

```diff
@@ -33,18 +47,32 @@ paper-button {
   margin-bottom: 24px;
   margin-left: 0px;
 }
+
 .drawer-icon {
   padding-right: 16px;
 }
+
 google-map {
   height: 600px;
 }
-#mainheader {
-  height: 100%;
-  background: #efefef;
-  overflow: auto;
-  -webkit-overflow-scrolling: touch;
+
+@media (prefers-color-scheme: dark) {
```

Comment:
I think it might be easiest to have some variables defined for the colors at the start of the file and then you can just use the color variables throughout (see https://stackoverflow.com/questions/56300132/how-to-override-css-prefers-color-scheme-setting)